tiflash/use-tiflash.md

```diff
@@ -277,11 +277,20 @@ explain select count(*) from customer c join nation n on c.c_nationkey=n.n_natio
 
 In the example execution plan, the `ExchangeReceiver` and `ExchangeSender` operators are included. The execution plan indicates that after the `nation` table is read, the `ExchangeSender` operator broadcasts the table to each node, the `HashJoin` and `HashAgg` operations are performed on the `nation` table and the `customer` table, and then the results are returned to TiDB.
 
-> **Note:**
->
-> By default, TiFlash does not support using the MPP mode in partitioned tables.
-
 TiFlash provides the following two global/session variables to control whether to use Broadcast Hash Join:
 
 - [`tidb_broadcast_join_threshold_size`](/system-variables.md#tidb_broadcast_join_threshold_count-new-in-v50-ga): The unit of the value is bytes. If the table size (in the unit of bytes) is less than the value of the variable, the Broadcast Hash Join algorithm is used. Otherwise, the Shuffled Hash Join algorithm is used.
 - [`tidb_broadcast_join_threshold_count`](/system-variables.md#tidb_broadcast_join_threshold_count-new-in-v50-ga): The unit of the value is rows. If the objects of the join operation belong to a subquery, the optimizer cannot estimate the size of the subquery result set, so the size is determined by the number of rows in the result set. If the estimated number of rows in the subquery is less than the value of this variable, the Broadcast Hash Join algorithm is used. Otherwise, the Shuffled Hash Join algorithm is used.
+
+> **Note:**
+>
+> TiFlash Computation layer does not support:
```

Comment:
```suggestion
> TiFlash computation layer does not support:
```

or just TiFlash doesn't support? I don't think a user is interested with what "computation layer" is.